Source/SG/SG.cs

```diff
@@ -137,9 +138,18 @@ public static void ConfigureInstrument(NIRfsg rfsgHandle, InstrumentConfiguratio
                         rfsgHandle.RF.LocalOscillator[loConfig.ChannelName].Source = RfsgLocalOscillatorSource.FromString(loConfig.Source);
                         break;
                     default: // default to automatic case
-                        rfsgHandle.RF.LOOutExportConfigureFromRfsa = RfsgLOOutExportConfigureFromRfsa.Enabled;
+                        string instrumentModel = rfsgHandle.Identity.InstrumentModel;
+                        if (instrumentModel == "PXIe-5830" || instrumentModel == "PXIe-5831")
+                        {
+                            rfsgHandle.RF.LOOutExportConfigureFromRfsa = RfsgLOOutExportConfigureFromRfsa.Disabled;

```

Comment:
This might error out on the 5830/31 since this property is only supported on 5840 (not sure). Maybe better just to reset it?

Reply:
@mdwhitten should be resolved in latest commit